testenv2/conf/config.yaml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
the spid testenv config should not be in this repository, it's a separate project!

Reply:
I added this file according to the structure which I currently found on the [io-backend](https://github.com/teamdigitale/io-backend/tree/master/testenv2/conf) repository. Should I then delete this file and add it to `.gitignore`?

Reply:
just delete the `testenv2` directory, it shouldn't be part of this project (do you understand why?)